scripts/ci_launch.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/bash -i
-set -e
+# set -e
+
+# Default GSW is cosmos
+GSW="cosmos"
+
+# Parse GSW selection
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --use-yamcs)
+      GSW="yamcs"
+      shift
+      ;;
+    --use-cosmos)
+      GSW="cosmos"
+      shift
+      ;;
+    *)
+      echo "Unknown option: $1"
+      echo "Usage: $0 [--use-cosmos | --use-yamcs]"
+      exit 1
+      ;;
+  esac
+done
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 source "$SCRIPT_DIR/env.sh"
```

Comment:
Should look to use the provided $DFLAGS and other commands instead of using those calls directly to enable future updates to support podman.